### PR TITLE
Update and rename background-and-history.md to README.md

### PR DESCRIPTION
**Pull request by ContentBrewmaster**
Making the new landing page under About us.

----
:books: Documentation preview :books:: https://civicactions-handbook--1140.org.readthedocs.build/en/1140/

### CODE REVIEW

**Comment by dmundra** [comment]
@ContentBrewmaster looks like the check found a broken link: https://github.com/CivicActions/guidebook/blob/ContentBrewmaster-About-Us-Readme/company-policies/new-hire-orientation/welcome.md?plain=1#L18

Also we want to update the redirect.yml file, specifically this line https://github.com/CivicActions/guidebook/blob/ContentBrewmaster-About-Us-Readme/.config/redirects.yml#L61. The change would be dropping the background-and-history from the `to_url`

```
- type: page
  from_url: /020-about-us/background-and-history/
  to_url: /about-civicactions/
```

**Comment by dmundra** [comment]
@ContentBrewmaster when we merge Owen's magic-mover changes it will do the changes I mentioned automatically so we should give that a try.

**Comment by ContentBrewmaster** [comment]
@dmundra should I still update the https://github.com/CivicActions/guidebook/blob/ContentBrewmaster-About-Us-Readme/company-policies/new-hire-orientation/welcome.md?plain=1#L18 or will Owen's update fix this?

**Comment by dmundra** [comment]
@ContentBrewmaster Owen's update will fix that. It is some nice wizardry.

File: .config/redirects.yml [hunk 1]
```diff
@@ -1,645 +1,483 @@
...
   to_url: /practice-areas/project-management/leave-requests-and-stepping-away/
-
-- type: page
-  from_url: /10-ux/services/design/content-modeling-guide/
+  type: page
+- from_url: /10-ux/services/design/content-modeling-guide/
   to_url: /practice-areas/design-and-research/content-modeling-guide/
+  type: page
+- from_url: /about-civicactions/background-and-history
+  to_url: /about-civicactions/README
```
Comment (dmundra) [inline .config/redirects.yml]: @grugnog looks like I found a small bug that is the from_url here needs a trailing slash for the redirect to work (I think looking at other examples). Here is the URL https://civicactions-handbook--1140.org.readthedocs.build/en/1140/about-civicactions/background-and-history/ that goes to the 404 page (the 404 is broken in previews).

Comment (grugnog) [reply .config/redirects.yml]: Good catch - thanks!
I added a fix to this branch (and rebased out the old fix) - can see if that does the trick.

Comment (dmundra) [reply .config/redirects.yml]: Thank you!

**Review by dmundra — APPROVED** [review]
The changes look good! Thanks @ContentBrewmaster and @grugnog

**Pull request merged**